app/transcribe/app_utils.py

```diff
@@ -192,6 +196,17 @@ def create_transcriber(
     global_vars.set_transcriber(t)
 
 
+def get_language_code(lang: str) -> str:
```

Comment:
I think this method is repeated. Not a big deal, can be taken care of in a subsequent PR.